mmv1/products/certificatemanager/api.yaml

```diff
@@ -212,4 +212,211 @@ objects:
+          base_url: '{{op_id}}'
+          wait_ms: 1000
+        result: !ruby/object:Api::OpAsync::Result
+          path: 'response'
+        status: !ruby/object:Api::OpAsync::Status
+          path: 'done'
+          complete: true
+          allowed:
+            - true
+            - false
+        error: !ruby/object:Api::OpAsync::Error
+          path: 'error'
+          message: 'message'
+      parameters:
+        - !ruby/object:Api::Type::String
+          name: 'name'
+          required: true
+          input: true
+          description: |
+            A user-defined name of the Certificate Map Entry. Certificate Map Entry 
+            names must be unique globally and match pattern 
+            'projects/*/locations/*/certificateMaps/*/certificateMapEntries/*'
+      properties:
+        - !ruby/object:Api::Type::String
+          name: 'description'
+          description: |
+            CertificateMapEntry is a list of certificate configurations,
+            that have been issued for a particular hostname
+        - !ruby/object:Api::Type::String
+          name: 'createTime'
+          output: true
+          description: |
+            Creation timestamp of a Certificate Map Entry. Timestamp in RFC3339 UTC "Zulu" format, 
+            with nanosecond resolution and up to nine fractional digits. 
+            Examples: "2014-10-02T15:01:23Z" and "2014-10-02T15:01:23.045123456Z".
+        - !ruby/object:Api::Type::Time
+          name: 'updateTime'
+          description: |
+            Update timestamp of a Certificate Map Entry. Timestamp in RFC3339 UTC "Zulu" format, 
+            with nanosecond resolution and up to nine fractional digits. 
+            Examples: "2014-10-02T15:01:23Z" and "2014-10-02T15:01:23.045123456Z".
+          output: true
+        - !ruby/object:Api::Type::KeyValuePairs
+          name: 'labels'
+          description: |
+            Set of labels associated with a Certificate Map Entry.
+            An object containing a list of "key": value pairs.
+            Example: { "name": "wrench", "mass": "1.3kg", "count": "3" }.
+        - !ruby/object:Api::Type::Array
+          name: 'certificates'
+          description: |
+            A set of Certificates defines for the given hostname.
+            There can be defined up to fifteen certificates in each Certificate Map Entry.
+            Each certificate must match pattern projects/*/locations/*/certificates/*.
+          item_type: Api::Type::String
+        - !ruby/object:Api::Type::Enum
+          name: 'state'
+          output: true
+          description: |
+            A serving state of this Certificate Map Entry.
+
+            The status is undefined.
+
+            The configuration is serving.
 
+            Update is in progress. Some frontends may serve this configuration.
+          values:
+            - :SERVING_STATE_UNSPECIFIFED
+            - :ACTIVE
+            - :PENDING
+          default_value: :ACTIVE
```

Comment:
ah! `output` and `default_value` set here. they can't both be set. if it's output-only, there won't be a default